src/pages/style.css

```diff
@@ -114,19 +114,19 @@ Mobile & tablet visibility helpers, specific to dev-dot.
 */
 @media (--dev-dot-mobile) {
   .g-hide-on-mobile {
-    display: none;
+    display: none !important;
```

Comment:
`!important` is _generally_ no good, in this case though it seems we need it.

This is maybe a good example of a caveat of using global helper classes, vs inlining these styles 🤷 For `.mobileMenuButton` specifically, if this style was inline, it might be more clear why this breaks (we compose and then basically override with `display: flex`).